models.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime
+
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy.dialects.postgresql import JSON
+
+db = SQLAlchemy()
+
+
+class CacheItem(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
```

Comment:
It's a linter warning but for consistency with database conventions (and the SQLAlchemy/Django docs) I've always used `id` for these columns. :)